SessionContext ITs are flaky

```
Error:  Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.889 s <<< FAILURE! - in com.vaadin.flow.quarkus.it.SessionContextIT
Error:  com.vaadin.flow.quarkus.it.SessionContextIT.httpSessionCloseDestroysSessionContext  Time elapsed: 1.157 s  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at vaadin.quarkus.production.tests@1.1-SNAPSHOT/com.vaadin.flow.quarkus.it.SessionContextIT.assertDestroyCountEquals(SessionContextIT.java:86)
	at vaadin.quarkus.production.tests@1.1-SNAPSHOT/com.vaadin.flow.quarkus.it.SessionContextIT.httpSessionCloseDestroysSessionContext(SessionContextIT.java:54)
```

```
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.473 s <<< FAILURE! - in com.vaadin.flow.quarkus.it.SessionContextCloseIT
Error:  com.vaadin.flow.quarkus.it.SessionContextCloseIT.vaadinSessionCloseDestroysSessionContext  Time elapsed: 1.459 s  <<< FAILURE!
java.lang.AssertionError: expected:<1> but was:<0>
	at vaadin.quarkus.production.tests@2.0-SNAPSHOT/com.vaadin.flow.quarkus.it.SessionContextCloseIT.assertDestroyCountEquals(SessionContextCloseIT.java:51)
	at vaadin.quarkus.production.tests@2.0-SNAPSHOT/com.vaadin.flow.quarkus.it.SessionContextCloseIT.vaadinSessionCloseDestroysSessionContext(SessionContextCloseIT.java:46)
```

--- a/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/AbstractCdiIT.java
+++ b/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/AbstractCdiIT.java
@@ -41,6 +41,17 @@ abstract public class AbstractCdiIT extends AbstractChromeIT {
         return findElement(By.id(id)).getText();
     }
 
+    protected void waitForCount(int expectedCount, String counter) {
+        waitUntil(driver -> {
+            try {
+                assertCountEquals(expectedCount, counter);
+                return true;
+            } catch (Exception ex) {
+                return false;
+            }
+        }, 1);
+    }
+
     protected void assertCountEquals(int expectedCount, String counter)
             throws IOException {
         Assert.assertEquals(expectedCount, getCount(counter));

--- a/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/SessionContextCloseIT.java
+++ b/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/SessionContextCloseIT.java
@@ -48,6 +48,6 @@ public class SessionContextCloseIT extends AbstractCdiIT {
 
     private void assertDestroyCountEquals(int expectedCount)
             throws IOException {
-        assertCountEquals(expectedCount, DESTROY_COUNT);
+        waitForCount(expectedCount, DESTROY_COUNT);
     }
 }

--- a/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/SessionContextIT.java
+++ b/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/SessionContextIT.java
@@ -83,7 +83,7 @@ public class SessionContextIT extends AbstractCdiIT {
 
     private void assertDestroyCountEquals(int expectedCount)
             throws IOException {
-        assertCountEquals(expectedCount, DESTROY_COUNT);
+        waitForCount(expectedCount, DESTROY_COUNT);
     }
 
     private static Logger getLogger() {

